django/thunderstore/api/cyberstorm/tests/test_package_listing_actions.py

```diff
@@ -54,6 +58,7 @@ def perform_package_listing_action_test(
     user_type: str,
     url: str,
     data: dict,
+    expected_status_code: dict = {},
```

Comment:
My previous comment was intended more towards fixing the type hint than changing the default value. I do remember that an empty dict as the default value is a footgun when multiple function calls mutate it, but I don't remember the details so IDK if it matters here.